qt/scientific_interfaces/test/ISISReflectometry/Runs/RunsPresenterTest.h

```diff
@@ -205,6 +205,30 @@ class RunsPresenterTest : public CxxTest::TestSuite {
     verifyAndClear();
   }
 
+  void testWarningGivenIfUnsavedBatchAutoreductionResumed() {
```

Comment:
Is it worth adding an inverse of this test i.e. that no warning is given if the option is disabled?